Sources/CoreFoundation/include/ForFoundationOnly.h

```diff
@@ -68,7 +68,10 @@ CF_EXPORT void *_Nonnull __CFSafelyReallocate(void * _Nullable destination, size
 CF_EXPORT void *_Nonnull __CFSafelyReallocateWithAllocator(CFAllocatorRef _Nullable, void * _Nullable destination, size_t newCapacity, CFOptionFlags options, void (^_Nullable reallocationFailureHandler)(void *_Nonnull original, bool *_Nonnull outRecovered));
 #endif
 
+_CF_EXPORT_SCOPE_BEGIN
 Boolean __CFAllocatorRespectsHintZeroWhenAllocating(CFAllocatorRef _Nullable allocator);
+_CF_EXPORT_SCOPE_END
```

Comment:
Will making this CF_EXPORT like the others also fix the issue?

Reply:
Good question! `CF_EXPORT` seems to expand into `extern` on Linux, not `extern "C"`:

https://github.com/apple/swift-corelibs-foundation/blob/a9ffa779b1fdaa767c9c6ea28acfca32f7efe460/Sources/CoreFoundation/include/CFBase.h#L139

I'm wondering if this is an oversight, I can tweak the `CF_EXPORT` macro to expand into `extern "C"` in C++ language mode, and then use `CF_EXPORT` for this function.

Reply:
In truth no one is supposed to be using CoreFoundation except Foundation.

Reply:
Usually our CF headers start with the macro the begin extern “C” but since this one is internal to Foundation it doesn’t have it.

Reply:
I think this header file is included by a public Foundation header, so it ends up being parsed when someone is including Foundation.

There are a few `_CF_EXPORT_SCOPE_BEGIN`/`_CF_EXPORT_SCOPE_END` blocks below in this file, so I thought that this is the preferred solution. But I'm happy to switch to `CF_EXPORT` if you think that's better.

Reply:
To avoid this turning into a tar pit of changes, let's just extend the begin/end to cover this 'group' of functions. Lines 60 - 75

Reply:
Sounds good, done.